### PR TITLE
intial refactor

### PR DESCRIPTION
**Pull request by rmessina1010**
Refactoring platform/testing/exportsFile, commenting out troublesome exports which caused test failures. These exports are still available via their own individual files, exportsFile.js serves as an otherwise convenient aggregate.

### CODE REVIEW

File: src/platform/testing/exportsFile.js [hunk 1]
```diff
@@ -8,39 +8,41 @@ export {
...
-  fillDate,
-} from './unit/schemaform-utils';
-
 export { testkit, sentryTransport } from './unit/sentry';
+
+// Passthrough for the following does not seem to be woking. For now,
+// use the corresponding path alias when importing.
+//
+// export {
+//   DefinitionTester,
```
Comment (acrollet) [inline src/platform/testing/exportsFile.js:22]: verified these are all exported.

File: src/platform/testing/exportsFile.js [hunk 2]
```diff
@@ -8,39 +8,41 @@ export {
...
+//   submitForm,
+//   getFormDOM,
+//   fillData,
+//   selectCheckbox,
+//   selectRadio,
+//   fillDate as fillSchemaform,
```
Comment (acrollet) [inline src/platform/testing/exportsFile.js:28]: verified `fillSchemaform` is not referenced elsewhere in the codebase.

File: src/platform/testing/exportsFile.js [hunk 3]
```diff
@@ -8,39 +8,41 @@ export {
...
+// } from './unit/schemaform-utils';
+
+// export {
+//   changeDropdown,
```
Comment (acrollet) [inline src/platform/testing/exportsFile.js:32]: verified these are all exported.

File: src/platform/testing/exportsFile.js [hunk 4]
```diff
@@ -8,39 +8,41 @@ export {
...
+//   setFetchJSONResponse,
+//   setFetchJSONFailure,
+//   setFetchBlobResponse,
+//   setFetchBlobFailure,
+//   resetFetch,
+//   mockApiRequest,
+//   mockMultipleApiRequests,
+//   createTestHistory,
+//   expect,
+//   fillDate as fillDateHelper,
```
Comment (acrollet) [inline src/platform/testing/exportsFile.js:44]: verified `fillDateHelper` isn't referenced elsewhere.

**Review by acrollet — APPROVED** [review]
LGTM!

**Review by rsmithadhoc — APPROVED** [review]

**Pull request merged**